Code/Editor/EditorViewportWidget.h

```diff
@@ -353,242 +221,173 @@ AZ_POP_DISABLE_DLL_EXPORT_BASECLASS_WARNING
 
-    void OnMenuResolutionCustom();
-    void OnMenuCreateCameraEntityFromCurrentView();
-    void OnMenuSelectCurrentCamera();
-
-    int OnCreate();
-    void resizeEvent(QResizeEvent* event) override;
-    void paintEvent(QPaintEvent* event) override;
-    void mousePressEvent(QMouseEvent* event) override;
-
-    // From a series of input primitives, compose a complete mouse interaction.
-    AzToolsFramework::ViewportInteraction::MouseInteraction BuildMouseInteractionInternal(
-        AzToolsFramework::ViewportInteraction::MouseButtons buttons,
-        AzToolsFramework::ViewportInteraction::KeyboardModifiers modifiers,
-        const AzToolsFramework::ViewportInteraction::MousePick& mousePick) const;
-    // Given a point in the viewport, return the pick ray into the scene.
-    // note: The argument passed to parameter **point**, originating
-    // from a Qt event, must first be passed to WidgetToViewport before being
-    // passed to BuildMousePick.
-    AzToolsFramework::ViewportInteraction::MousePick BuildMousePick(const QPoint& point);
-
-    bool event(QEvent* event) override;
-    void OnDestroy();
-
-    bool CheckRespondToInput() const;
-
-    // AzFramework::InputSystemCursorConstraintRequestBus
-    void* GetSystemCursorConstraintWindow() const override;
-
-    void BuildDragDropContext(AzQtComponents::ViewportDragContext& context, const QPoint& pt) override;
-
-private:
-    void SetAsActiveViewport();
-    void PushDisableRendering();
-    void PopDisableRendering();
-    bool IsRenderingDisabled() const;
-    AzToolsFramework::ViewportInteraction::MousePick BuildMousePickInternal(const QPoint& point) const;
-
-    void RestoreViewportAfterGameMode();
-    void UpdateCameraFromViewportContext();
-
-    double WidgetToViewportFactor() const
-    {
-#if defined(AZ_PLATFORM_WINDOWS)
-        // Needed for high DPI mode on windows
-        return devicePixelRatioF();
-#else
-        return 1.0f;
-#endif
-    }
-
-    void BeginUndoTransaction() override;
-    void EndUndoTransaction() override;
-
-    void UpdateCurrentMousePos(const QPoint& newPosition);
-    void UpdateScene();
-
+    // Shim for QtViewport, which used to be responsible for visibility queries in the editor,
+    // these are now forwarded to EntityVisibilityQuery
     AzFramework::EntityVisibilityQuery m_entityVisibilityQuery;
 
+    // Handlers for grid snapping/editor event callbacks
     SandboxEditor::GridSnappingChangedEvent::Handler m_gridSnappingHandler;
     AZStd::unique_ptr<SandboxEditor::EditorViewportSettingsCallbacks> m_editorViewportSettingsCallbacks;
 
+    // Used for some legacy logic which lets the widget release a grabbed keyboard at the right times
+    // Unclear if it's still necessary.
     QSet<int> m_keyDown;
 
+    // State for ViewportFreezeRequestBus, currently does nothing
     bool m_freezeViewportInput = false;
 
+    // This widget holds a reference to the manipulator manage because its responsible for drawing manipulators
```

Comment:
nit: manage -> manager (also customarily for these sorts of comments in a header we use `//!` style and all end with a period (not a deal breaker but just a style to be aware of that's followed in most other places)

We actually have finally published the coding standards and API ref guidelines here too! - https://github.com/o3de/sig-core/tree/main/governance